validator/node/node.go

```diff
@@ -628,6 +624,45 @@ func proposerSettings(cliCtx *cli.Context, db iface.ValidatorDB) (*validatorServ
 	return vpSettings, nil
 }
 
+func handleNoProposerSettingsFlagsProvided(cliCtx *cli.Context,
+	db iface.ValidatorDB,
+	builderConfigFromFlag *validatorServiceConfig.BuilderConfig) (*validatorServiceConfig.ProposerSettings, error) {
+	// checks db if proposer settings exist if none is provided.
+	settings, err := db.ProposerSettings(cliCtx.Context)
+	if err == nil {
```

Comment:
Should you handle the error? At least log it to the user.

Reply:
I was debating this. I probably should log something though

Reply:
yeah, i believe that all errors should be handled somehow and usually that means surfacing it to the user somehow. Perhaps as a warning or error level message.

Reply:
added some additional logs https://github.com/prysmaticlabs/prysm/pull/12456/commits/4dd636fb11b80b29a167e74eb40379253f9e928d based on the feedback. I set it as a log info instead of a warn. this is because users can also choose to not provide anything + not have anything in the db and completely depend on the beacon node's suggested fee recipient settings. this avoids the API calls. let me know if you think a warn would be better in this case.

Reply:
I take that back, changed it to warn anyways since it didn't like the err outside of "WithError"